RandomCatsTests/RandomCatsTests.swift

```diff
@@ -1,14 +1,14 @@
 //
-//  RandomDogsTests.swift
```

Comment:
we could come up with some unit tests for this project, probably on the API stuff mostly :]

Reply:
Yes, that would be great. Let's meet sometime for a unit test and UI test lesson. Thank you so much!